app/models/season.rb

```diff
@@ -11,5 +11,6 @@
 
 class Season < ActiveRecord::Base
   has_many :episodes
-  has_and_belongs_to_many :seasons
+  has_many :contestants_seasons, class_name: 'ContestantsSeasons'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.